content/efcore/getting-started/samples/manipulating-entities/updating-untracked-entities/Program.cs

```diff
@@ -7,21 +7,21 @@ public static void Main()
     {
         using (var context = new LibraryContext())
         {
-            var untrackedBook = new Book()
-            {
-                Id = 1,
-                Title = "Frankenstein: or, The Modern Prometheus",
-                Genre = "Science Fiction",
-                PublicationYear = 1818
-            };
-
-            context.Books.Update(untrackedBook);
-            context.SaveChanges();
+            var untrackedBook = new Book() 
+			{ 
+				Id = 1, 
+				Title = "To the Lighthouse",
+				Genre = "Literary",
+				PublicationYear = 1927 
+			};
+			
+			context.Books.Update(untrackedBook); 
+			context.SaveChanges();
 
             var updatedBook = context.Books
                 .Single(b => b.Id == 1);
 
-            Console.WriteLine("{0}: {1}, {2} {3}", updatedBook.Id, updatedBook.Title, updatedBook.Genre, updatedBook.PublicationYear);
+            Console.WriteLine("Updated Book -\n Id: {0}\n Title: {1}\n Genre: {2}\n Publication Year: {3}", updatedBook.Id, updatedBook.Title, updatedBook.Genre, updatedBook.PublicationYear);
```

Comment:
Use multiple `Console.WriteLine` statements instead of the newlines. It will make the code more readable.